lib/components/ReliefActivitiesMenu/ActivityButton.dart

```diff
@@ -20,53 +23,88 @@ class _ActivityButtonState extends State<ActivityButton> {
   @override
   Widget build(BuildContext context) {
     Color color;
+    LinearGradient grad = AppConstants.getGradByMood(widget.activity.mood);
     const double radius = 10;
     double deviceHeight = MediaQuery.of(context).size.height;
-    double buttonMinHeight = deviceHeight * 0.125;
+
+    // KNOWN BUG: Favorites button overflows. Currently fixed by making the cards a lil taller
+    double buttonMinHeight = deviceHeight * 0.13;
     setState(() {
       iconData = widget.activity.favorite ? Icons.star : Icons.star_outline;
     });
 
     color = AppConstants.getColorByMood(widget.activity.mood);
-
+    
     ElevatedButton button = ElevatedButton(
         onPressed: () {
           Navigator.push(
               context,
               MaterialPageRoute(
                   builder: (context) => ReliefScreen(data: widget.activity)));
         },
-        child: Align(
-          child: Column(children: [
-            const Spacer(),
-            Text(widget.activity.activityName,
-                style: const TextStyle(color: AppColors.white)),
-            IconButton(
-                onPressed: () {
-                  // Set favorite
-                  widget.activity.toggleActivityFavorite();
-                  setState(() {
-                    iconData = iconData == Icons.star
-                        ? Icons.star_outline
-                        : Icons.star;
-                  });
-                },
-                icon: Icon(
-                  iconData,
-                  size: 20,
-                )),
-            const Spacer(),
-          ]),
-          alignment: Alignment.center,
-        ),
         style: ButtonStyle(
-            backgroundColor: MaterialStateProperty.all<Color>(color),
+            //sets background color of buttons to white
+            backgroundColor: MaterialStateProperty.all<Color>(
+                Color.fromRGBO(255, 255, 255, 1)),
```

Comment:
Can't you use the color from our App Constants?

Reply:
Yes